src/mainviews/job-detail.tsx

```diff
@@ -1,13 +1,398 @@
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('Job Definition Id')}
+                  defaultValue={job?.job_definition_id ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <FormLabel component="legend">{trans.__('Tags')}</FormLabel>
+                {job?.tags &&
+                  job?.tags.map(tag => (
+                    <TextFieldStyled
+                      label={trans.__('Tag')}
+                      defaultValue={tag}
+                      InputProps={{
+                        readOnly: true
+                      }}
+                    />
+                  ))}
+                <FormLabel component="legend">
+                  {trans.__('Email Notifications')}
+                </FormLabel>
+                <TextFieldStyled
+                  label={trans.__('Job Definition Id')}
+                  defaultValue={job?.timeout_seconds?.toString() ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('max_retries')}
+                  defaultValue={job?.max_retries?.toString() ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('min_retry_interval_millis')}
+                  defaultValue={
+                    job?.min_retry_interval_millis?.toString() ?? ''
+                  }
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('retry_on_timeout')}
+                  defaultValue={job?.retry_on_timeout?.toString() ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('start_time')}
+                  defaultValue={job?.start_time?.toString() ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('output_filename_template')}
+                  defaultValue={job?.output_filename_template ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+
+                <TextFieldStyled
+                  label={trans.__('url')}
+                  defaultValue={job?.url ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('status_message')}
```

Comment:
```suggestion
                  label={trans.__('Status message')}
```

Shouldn't this be displayed right after "Status"?

Reply:
As it is now, `status_message` is always `null` in job object that server/API returns based on `jobId` while `status` contains actual status enum / string
![Screen Shot 2022-09-26 at 1 56 15 PM](https://user-images.githubusercontent.com/26686070/192378976-bb3bc457-f7b9-4344-8be6-979e2f0d0c7a.png)

Reply:
status_message removed completely as it's not implemented